spec/rubocop/cop/metrics/utils/abc_size_calculator_spec.rb

```diff
@@ -42,7 +42,28 @@ def method_name
             end
           end
         RUBY
-        expect(described_class.calculate(node)).to be_within(0.001).of(8.77)
+        expect(described_class.calculate(node)).to be_within(0.001).of(9.17)
+      end
+    end
+
+    context '2 assignments, 9 branches, 5 conditions' do
+      it 'returns 10.49' do
+        node = parse_source(<<-RUBY.strip_indent).ast
+          def method_name
+            a = b ? c : d
+            if a < b
+              a
+            else
```

Comment:
What about `elsif`? How does that count now?

Reply:
Great question. I don't think there is any change in how `elsif`. `if`, `elsif`, and `else` each increment the condition count by 1. Since `elsif` is an `if` node, it was being counted before. `else` was not incrementing the count because it is not a standalone node.

Based on the wiki article, I'm not entirely sure how `else if` would be counted in other languages. The list of given keywords for other languages includes ‘else’, ‘case’, ‘default’, ‘?’, ‘try’, ‘catch’. There is no mention of `if` which seems wrong. I was thinking `if` was included`. With that assumption in mind, would `else if` increment count by 2 in other languages? Since the implementation is language specific, I think `if`, `elsif`, and `else` should each increment by 1.

Reply:
Maybe you could add a spec example demonstrating `elsif`? Or even two examples showing that `elsif` and `else if` count the same.

Reply:
That seems OK.